src/main/java/pl/edu/agh/iosr/cdm/Proposal.java

```diff
@@ -2,17 +2,28 @@
 
 import lombok.Builder;
 import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
 @Data
 @Builder
+@Entity
 public class Proposal {
-    private long id;
-    private int value;
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
```

Comment:
Czy jak bedziemy generowac id to nam go czasem nie nadpisze?
To id to identyfikator requestu ustalony przez proposera, chyba trzeba dołożyć inne id na potrzeby bazy.